added artist name on hover for album search page and album title on top-albums section of artist search page

meeday opened the pull request.

meeday committed:
added artist name on hover for album search page

Goldenaries658 committed:
Prettier run

Goldenaries658 commented on assets/script.js:
```diff
@@ -116,6 +116,10 @@ function getAlbumInfo() {
         $("#album-pic").attr("src", icon);
         $("#summaryHeading").text(albumName);
+        $("#album-pic").attr("title", response.album.artist);
+        $("a.img").each(function () {
+          $(this).attr("title", $(this).find("img").attr("title"));
+        });
```
Comment exactly what this is doing as it's not the most readable line

meeday replied:
bug fixed

Goldenaries658 reviewed (changes requested):
Looking good so far just need to fix this visual bug:  
![image](https://user-images.githubusercontent.com/59972622/81177203-32511200-8f9e-11ea-9076-63bca52596ac.png)

meeday committed:
visual bug fixed

meeday committed:
described js code for hover function

meeday committed:
image title on hover for top-albums

commented on the function used and added title attributes to the top-albums section in artist search page

Goldenaries658 committed:
Formatting

meeday committed:
improved the titles consistency

Goldenaries658 committed:
Ran prettier
